### PR TITLE
Add beta warning message and IP whitelisting warning

### PR DESCRIPTION
**Pull request by thild42**
Add beta disclaimer on top of the doc (like in Notebooks Spark).
Add a warning to whitelist the IP address of the Data Integration platform on the managed databases.

### CODE REVIEW

**Comment by thild42** [comment]
I leave the PR in draft until the Data Integration platform is redeployed (should happen this week) and actually uses the IP mentioned in the documentation.

File: pages/platform/data-integration/guide_01_cli_installation/guide.de-de.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ routes:
 updated: 2023-06-27
```
Comment (OVH-Team-Guides) [inline pages/platform/data-integration/guide_01_cli_installation/guide.de-de.md]: Can you please update the date accordingly ?

File: pages/platform/data-integration/guide_01_cli_installation/guide.de-de.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ routes:
...
 ---
 
+> [!primary]
+> The Data Integration product is in `beta`. During the beta-testing phase, the infrastructure’s availability and performances are not guaranteed. Please do not use this service for applications that are in production, while this phase is not complete.
```
Comment (OVH-Team-Guides) [inline pages/platform/data-integration/guide_01_cli_installation/guide.de-de.md]:
```suggestion
> The Data Integration product is in `beta`. During the beta-testing phase, the infrastructure’s availability and performances are not guaranteed. Please do not use this service for applications that are in production while this phase is not finished.
```

File: pages/platform/data-integration/guide_02_cli_commands_reference/guide.de-de.md [hunk 3]
```diff
@@ -30,7 +33,12 @@ ovhdata-cli di source-connector list
 
 You should see a list of source connectors with their `name`, `identifier` and `version`.
 
-Let's create a PostgreSQL source. We can either provide all the required parameters on the command line or just the name of the source. If one or multiple parameters are missing,the CLI will help us fill them in interactively:
+Let's create a PostgreSQL source targetting a managed database hosted in OVHcloud Public Cloud.
```
Comment (OVH-Team-Guides) [inline pages/platform/data-integration/guide_02_cli_commands_reference/guide.de-de.md]:
```suggestion
Let's create a PostgreSQL source targeting a managed database hosted on OVHcloud Public Cloud.
```

File: pages/platform/data-integration/guide_02_cli_commands_reference/guide.de-de.md [hunk 4]
```diff
@@ -30,7 +33,12 @@ ovhdata-cli di source-connector list
...
+
+> [!primary]
+> Before creating this source, make sure the Data Integration platform IP address `188.165.35.170` is whitelisted on the managed database service.
+
+We can either provide all the required parameters on the command line or just the name of the source. If one or multiple parameters are missing,the CLI will help us fill them in interactively:
```
Comment (OVH-Team-Guides) [inline pages/platform/data-integration/guide_02_cli_commands_reference/guide.de-de.md]:
```suggestion
We can either provide all the required parameters on the command line or just the name of the source. If one or multiple parameters are missing, the CLI will help us fill them in interactively:
```

**Comment by thild42** [comment]
All suggestion were merged.
PR still in draft waiting for a redeployment of the platform in the beginning of next week.

**Comment by OVH-Team-Guides** [comment]
Closing this PR as it is now replaced by https://github.com/ovh/docs/pull/5113

**Pull request closed without merging**